isis/src/qisis/objs/CubeViewport/CubeViewport.cpp

```diff
@@ -1669,7 +1670,7 @@ namespace Isis {
 
 
   //! Return the gray band stretch
-  Stretch CubeViewport::grayStretch() const {
+  CubeStretch CubeViewport::grayStretch() const {
```

Comment:
There are a lot of other instances of Stretch in this file that aren't changed. For example, just after this there is CubeViewport::redStretch(), CubeViewport::greenStretch(), and CubeViewport::blueStretch(), which are all left as 'Stretch'. So I am curious why certain Typecasts are left while these ones are changed.

Reply:
This was an oversight because originally this was just designed to work with gray stretches. Thanks for catching this and pointing it out!